faiss/IndexAdditiveQuantizerFastScan.h

```diff
@@ -69,6 +69,8 @@ struct IndexAdditiveQuantizerFastScan : IndexFastScan {
             idx_t k,
             float* distances,
             idx_t* labels) const override;
+
+    void sa_decode(idx_t n, const uint8_t* bytes, float* x) const override;
```

Comment:
This deserves a bit of documentation because the bytes are non-contiguous in the index memory. Could you say that this takes a packed representation?